core/src/main/java/cz/xtf/core/config/WaitingConfig.java

```diff
@@ -1,11 +1,22 @@
 package cz.xtf.core.config;
 
+import org.slf4j.event.Level;
+
 public class WaitingConfig {
 	public static final String WAITING_TIMEOUT = "xtf.waiting.timeout";
 
 	private static final String WAITING_TIMEOUT_DEFAULT = "180000";
 
+	public static final String WAITING_LOG_LEVEL = "xtf.waiting.log.level";
```

Comment:
Can you group the fields first by properties and then by default values? I find that a more consistent with other classes and easier to read.

Reply:
Good point, done